crates/nu-command/src/conversions/into/binary.rs

```diff
@@ -224,10 +197,7 @@ pub fn action(input: &Value, _args: &Arguments, span: Span) -> Value {
                 }
             };
 
-            Value::Binary {
-                val: val.to_vec(),
-                span,
-            }
+            Value::binary(val.to_vec(), span)
```

Comment:
I think this should be `(val.to_vec(), val_span)`

Reply:
Oh，thanks for the good catch!